setup.cfg

```diff
@@ -1,2 +1,8 @@
 [flake8]
 ignore=E501
+
+[tool:pytest]
+python_files=test_*.py
+addopts=-n=auto --verbose -r=a --junit-xml=results/junit.xml
```

Comment:
We're not currently running these tests in parallel in Travis CI, so I wonder if we should avoid introducing this (we should probably remove the pytest-xdist dependency too, if there's an issue with running them in parallel).